registry/storage/driver/gcs/gcs.go

```diff
@@ -48,8 +48,8 @@ import (
 const driverName = "gcs"
 const dummyProjectID = "<unknown>"
 
-// driverParameters is a struct that encapsulates all of the driver parameters after all values have been set
-type driverParameters struct {
+// DriverParameters is a struct that encapsulates all of the driver parameters after all values have been set. Create a new one of these with NewDriverParameters
```

Comment:
And https://github.com/deis/distribution/pull/10/commits/7fb48650b08c7598c26077c901840d1a4631d21f